src/components/screens/votingQueue/result/result.js

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect } from 'react';
+
+import { removeSearchParamsFromUrl } from '../../../../utils/searchParams';
+import LiskAmount from '../../../shared/liskAmount';
+import Box from '../../../toolbox/box';
+import BoxFooter from '../../../toolbox/box/footer';
+import { PrimaryButton } from '../../../toolbox/buttons';
+import ToggleIcon from '../toggleIcon';
+import TransactionResult from '../../../shared/transactionResult';
+
+import styles from './styles.css';
+
+const unlockTime = 5;
```

Comment:
If the user self-vote themself the unlock time is around a month. We should have that check somewhere.

Reply:
We decided to use the correct unlock times in another PR. I didn't have enough understanding of the task.